.lycheeexclude

```diff
@@ -0,0 +1,166 @@
+# Local or predefined end points
+http://localhost
+https://localhost
+http://127.0.0.1/
+https://127.0.0.1/
+http://127.0.0.1:10002/bar 
+http://127.0.0.1:10002/
+http://opensearch
+https://opensearch
+https://opensearch-dashboards
+http://opensearch-dashboards
+https://opensearch.internal.org/
+https://maps.search-services.aws.a2z.com/
+https://tiles.maps.search-services.aws.a2z.com/ 
+
+# Dummy urls in tests
+http://domain.
+http://www.domain.
+http://somehost
+https://somehost
+http://some.host
+https://some-host.com/
+https://other.some-host.com/
+http://test:user@somehost/
+https://some.another.host/
+http://noone.nowhere.none/
+http://bar
+http://foo
+http://test.com/
+https://files.foobar/
+https://tiles.foobar/
+https://1.1.1.1:9200/
+http://192.168.1.1:1234/
+http://9.8.7.6/
+http://1.2.3.4/
+http://8.8.8.8/
+https://path.to/
+https://example.com/
+http://example.com/
+https://example.org/
+http://some-url/
+http://buildurl/
+https://dryrun/
+https://url/
+http://url/
+http://notfound.svg/ 
+https://validurl/
+https://myopensearch-dashboardsdomain.com
+http://myopensearch-dashboardsdomain.com
+https://other-opensearch-dashboards.external:8080/
+http://myotherdomain.com:5601/
+http://myopensearch-dashboardsdomain.com:5601/
+https://your-cdn-host.com/
+http://not-your-opensearch-dashboards.com/
+http://www.mysite.com/
+http://myserver.mydomain.com:5601/
+https://elastic:changeme@myCloudInstance:9200
+https://myexternaldep.com/
+http://notlocalhost/
+http://site.com/
+http://node-b/
+http://node-a:9200/
+https://node-c/
+https://elsewhere
+https://opensearch:changeme@example.com:9200
+http://elastic:changeme@localhost:61141/
```

Comment:
origin of `elastic` should be replaced or removed

Reply:
I will replace with some dummy url like `https://example.com/` since they are just hardcoded string in test files